### ISSUE TITLE
accessibility issues with the pihole web-ui

### ISSUE THREAD

**Issue by mhussaincov** [pi-hole/pi-hole#3192]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [] The issue I am reporting can be *replicated*.
- [] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**
1
`{Replace this with a number from 1 to 10. 1 being not familiar, and 10 being very familiar}`

---
**Expected behaviour:**
I use a screen reader orca on debian linux, the parts of the web-ui that are not accessible are the folowing 
Total queries over last 24 hours
after that is spoken by orca, the graphic part is not accessible,
all that orca the screen reader will tell me is canvus clickable.

nore is the part write after this bit.
Client activity over last 24 hours
the same is  repeated.

`{A detailed description of what you expect to see}`
would it be possible to have a text version of the graphical parts of the webui in text format for people like me who don't have site?
simply a text version of the pieces of information that is reprozented by the graphics?
e.g the urls accessed etc?

**Actual behaviour:**
the parts stated above are not accessible to orca the screen reader.
`{A detailed description and/or screenshots of what you do see}`

**Steps to reproduce:**
1.
launch orca from your linux distribution of choise most distributions have it pre installed.
2.
go to your pihole admin interface and log in.
3.
navigate to
Client activity over last 24 hours
and down arrow once.
you will hear no useful information after that 

this happens in the firefox browser and chromium, one

`{Detailed steps of how we can reproduce this}`

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

`{Alphanumeric token}`

**Troubleshooting undertaken, and/or other relevant information:**

I tryed chrome and firefox both with the same reaction.
`{Steps of what you have done to fix this}`
